src/keyboard/textinput.js

```diff
@@ -90,7 +90,8 @@ TextInput= function(parentNode, host) {
             text.setAttribute("aria-roledescription", nls("text-input.aria-roledescription", "editor"));
             if(host.session) {
                 var row =  host.session.selection.cursor.row;
-                text.setAttribute("aria-label", nls("text-input.aria-label", "Cursor at row $0", [row + 1]));
+                var arialLabel = `${host.$textInputAriaLabel}, ${nls("text-input.aria-label", "Cursor at row $0", [row + 1])}`;
```

Comment:
should we hide "," if there is textInputAriaLabel is not there? Basically to cater for the initial value.

Reply:
yeah that's fair, changed